features/step_definitions/main_structure_steps.rb

```diff
@@ -76,6 +85,28 @@
   step("I wait for progress")
 end
 
+Then (/^I make sure "([^\"]*)" is not present$/) do |text|
+  if element_exists("* {text CONTAINS[c] '#{text}'}")
+    fail("Item %s should have been deleted" % [text])
+  end
+end
+
+Then (/^I press "([^\"]*)" button$/) do |text|
+  tap_when_element_exists("* {text CONTAINS[c] '#{text}'}'")
+end  
+
+Then (/^I look for "([^\"]*)"$/) do |text|
+  check_element_exists("* {text CONTAINS[c] '#{text}'}")
+end  
+
+Then (/^I flip to landscape$/) do 
```

Comment:
Would be good to have this command consistent with the `Rotate Portrait` command name